src/SFA.DAS.ApprenticeCommitments/DTOs/ApprenticeshipDtoMapping.cs

```diff
@@ -57,8 +60,45 @@ private static ApprenticeshipDto MapApprenticeshipAndRevisionToApprenticeshipDto
                 ApprovedOn = latest.CommitmentsApprovedOn,
                 ChangeOfCircumstanceNotifications = apprenticeship.ChangeOfCircumstanceNotifications,
                 StoppedReceivedOn = latest.StoppedReceivedOn,
-                HasBeenConfirmedAtLeastOnce = apprenticeship.ApprenticeshipHasPreviouslyBeenConfirmed
+                HasBeenConfirmedAtLeastOnce = apprenticeship.ApprenticeshipHasPreviouslyBeenConfirmed,
+                Revisions = GetRevisions(apprenticeship.Revisions),
```

Comment:
The Timeline DTO is only needed when you get a confirmed apprenticeship. It's definitely not needed when we get the latest Apprenticeship